WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -59,6 +59,37 @@ struct PointOfSaleDashboardView: View {
                 }
             }
         })
+        .posModal(isPresented: $viewModel.showExitPOSModal, fixedWidth: false, fixedHeight: false, content: {
+            VStack(spacing: 0 ) {
+                HStack {
+                    Spacer()
+                    Button {
+                        viewModel.showExitPOSModal = false
+                    } label: {
+                        Image(PointOfSaleAssets.xClose.imageName)
+                            .frame(width: Constants.closeIconSize, height: Constants.closeIconSize)
+                            .foregroundColor(Color.posTertiaryTexti3)
+                    }
+                }
+                Text("Exit Point of Sale mode?")
+                    .font(.posModalTitle)
+                    .padding(.bottom)
+                Text("Any orders in progress will be lost.")
+                    .font(.posModalBody)
+                    .padding(.bottom)
+                    .padding(.bottom)
+                    .padding(.bottom)
+                Button {
+                    presentationMode.wrappedValue.dismiss()
```

Comment:
```suggestion
                    dismiss()
```
This is pre-existing, but it would be nice to shift to using:

    @Environment(\.dismiss) private var dismiss

for dismissal, as it's a bit more readable.